src/components/pagination/pagination.md

```diff
@@ -1,31 +1,70 @@
 Simple, few pages (default):
 
-    <div>
-      <Pagination selected={1} total={50} changeHandler={() => {}} />
-    </div>
+    const React = require('react');
+    const { HashRouter } = require('../../../node_modules/react-router-dom');
 
-Advanced:
+    class PaginationExample1 extends React.PureComponent {
+      constructor(props) {
+        super(props);
+        this.urlGenerator = this.urlGenerator.bind(this);
+      }
 
-    <div>
-      <Pagination selected={2} total={100} changeHandler={() => {}} />
-    </div>
+      urlGenerator(pageNumber) {
+        return `?page=${pageNumber}`
+      }
+
+      render() {
+        return (
+          <HashRouter>
```

Comment:
have you tried `MemoryRouter` instead? I believe Hash based router will conflict with styleguidist